plugins/cxaudio/cxaudio.quirk

```diff
@@ -0,0 +1,23 @@
+# ThinkPad TBT3 Gen 2 dock
+[DeviceInstanceId=USB\VID_17EF&PID_3083]
+Guid = CXAUDIO\CX2098X
+
+# ThinkPad USB-C Gen 2 dock
+[DeviceInstanceId=USB\VID_17EF&PID_XXXX]
```

Comment:
XXXX?

Reply:
Will fix!

Reply:
I'm awaiting the PID from Lenovo, but have commented this out in the meantime.